src/dscanner/analysis/objectconst.d

```diff
@@ -123,7 +112,7 @@ unittest
 					return 1;
 				}
 
-				const hash_t toHash() // ok
+				immutable hash_t toHash() // ok
```

Comment:
Why is this changed?

Reply:
The functionality checks if a function is either const immutable or inout. The unittest covered only const functions and I thought it would be good to have an example for each(const, immutable etc...)